awsmp/_driver.py

```diff
@@ -205,6 +205,36 @@ def _filter_instance_types(product_id: str, changeset):
     return changeset
 
 
+def _get_offer_id_from_listing(product_id: str):
```

Comment:
why do we add this and don't use it at all?

Reply:
it will get used soon :joy: just not in this PR but would like to automate update_instance_type without getting separate offer id